source/intro_release_notes/release_notes/whats_new.rst

```diff
@@ -9,6 +9,10 @@ OpenNebula Core
 - **Update hashing algorithm**, now passwords and login tokens are hashed using sha256 instead of sha1. Also csrftoken is now hashed with SHA256 instead of MD5
 - `FILTER is now a VM_RESTRICTED attribute <https://github.com/OpenNebula/one/issues/3092>`__.
 
+Storage
+--------------------------------------------------------------------------------
+- `Allows user to modify block size for dd commands used for Ceph, Fs and LVM datastore drivers <https://github.com/OpenNebula/one/issues/2808>`_.
```

Comment:
Also we need to updated the associated driver files. Eg. the ceph, fs, fs_lvm guides showing the new conf attribute and default value.